### ISSUE TITLE
Accessibility improvements - home screen

### ISSUE THREAD

**Issue by MillieMD** [MillieMD/revs-cocktail-quiz#5]
Accessibility issues found from [WAVE](https://wave.webaim.org/report#/https://milliemd.github.io/revs-cocktail-quiz/):

- Missing document language
- 2-4-1 limiter checkbox is missing label
